DEBUG - 2025-09-30 03:06:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:07:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:07:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:07:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:07:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:07:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:07:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:07:24 --> mysqli_sql_exception: Unknown column 'teacher_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:07:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'teacher_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `teacher_id` = :teacher_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:07:24 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'teacher_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `teacher_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `teacher_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `teacher_id` = :teacher_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:07:24 --> [Caused by] mysqli_sql_exception: Unknown column 'teacher_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `teacher_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `teacher_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `teacher_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `teacher_id` = :teacher_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:15:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:15:37 --> mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:15:37 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:15:37 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:15:37 --> [Caused by] mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:15:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:15:39 --> mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:15:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:15:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:15:39 --> [Caused by] mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:15:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:15:39 --> mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:15:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:15:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:15:39 --> [Caused by] mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(139): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:17:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:17:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:17:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:17:42 --> mysqli_sql_exception: Unknown column 'courses.name' in 'field list' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:17:42 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:17:42 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:17:42 --> [Caused by] mysqli_sql_exception: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:18:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:18:47 --> mysqli_sql_exception: Unknown column 'courses.name' in 'field list' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:18:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:18:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:18:47 --> [Caused by] mysqli_sql_exception: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:18:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:18:47 --> mysqli_sql_exception: Unknown column 'courses.name' in 'field list' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:18:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:18:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:18:47 --> [Caused by] mysqli_sql_exception: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(151): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:36:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:36:58 --> mysqli_sql_exception: Unknown column 'courses.name' in 'field list' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:36:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:36:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:36:58 --> [Caused by] mysqli_sql_exception: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:37:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:37:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:37:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:37:13 --> mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:37:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:37:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:37:13 --> [Caused by] mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:39:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:39:17 --> mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:39:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:39:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:39:17 --> [Caused by] mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:39:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:39:18 --> mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:39:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:39:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:39:18 --> [Caused by] mysqli_sql_exception: Unknown column 'instructure_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `instructure_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `instructure_id` = :instructure_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:44:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:44:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:44:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:44:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:44:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:44:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:44:32 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:44:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:44:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:44:32 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:44:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:44:32 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:44:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:44:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:44:32 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `courses`
WHERE `user_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `courses`
WHERE `user_id` = :user_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(138): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:51:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:51:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:52:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:52:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:52:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:52:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:52:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:52:16 --> mysqli_sql_exception: Unknown column 'courses.name' in 'field list' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:52:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:52:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:52:16 --> [Caused by] mysqli_sql_exception: Unknown column 'courses.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses`.`name`
FROM `enrollments`
JOIN `courses` ON `courses`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:53:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:53:38 --> mysqli_sql_exception: Table 'lms_herda.course' doesn't exist in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `course`...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `course`...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `course`...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `course`...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:53:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_herda.course' doesn't exist
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `course`.`name`
FROM `enrollments`
JOIN `course` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:53:38 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'lms_herda.course' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `course`.`name`
FROM `enrollments`
JOIN `course` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `course`.`name`
FROM `enrollments`
JOIN `course` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `course`.`name`
FROM `enrollments`
JOIN `course` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:53:38 --> [Caused by] mysqli_sql_exception: Table 'lms_herda.course' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `course`.`name`
FROM `enrollments`
JOIN `course` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `course`.`name`
FROM `enrollments`
JOIN `course` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `course`.`name`
FROM `enrollments`
JOIN `course` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `course`.`name`
FROM `enrollments`
JOIN `course` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:54:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 03:54:29 --> mysqli_sql_exception: Unknown column 'course.name' in 'field list' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `course`...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `course`...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `course`...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `course`...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 03:54:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'course.name' in 'field list'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `course`.`name`
FROM `enrollments`
JOIN `courses` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:54:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'course.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `course`.`name`
FROM `enrollments`
JOIN `courses` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `course`.`name`
FROM `enrollments`
JOIN `courses` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `course`.`name`
FROM `enrollments`
JOIN `courses` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 03:54:29 --> [Caused by] mysqli_sql_exception: Unknown column 'course.name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `course`.`name`
FROM `enrollments`
JOIN `courses` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `course`.`name`
FROM `enrollments`
JOIN `courses` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `course`.`name`
FROM `enrollments`
JOIN `courses` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `course`.`name`
FROM `enrollments`
JOIN `courses` ON `course`.`id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(146): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 03:55:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:56:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:56:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:56:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:59:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:59:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:59:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 03:59:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-09-30 04:00:11 --> mysqli_sql_exception: Unknown column 'courses_name' in 'field list' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Auth.php(150): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
#6 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-09-30 04:00:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses_name' in 'field list'
[Method: GET, Route: auth/dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses_name`
FROM `enrollments`
JOIN `courses` ON `courses_id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 2 APPPATH\Controllers\Auth.php(150): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 04:00:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'courses_name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses_name`
FROM `enrollments`
JOIN `courses` ON `courses_id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses_name`
FROM `enrollments`
JOIN `courses` ON `courses_id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses_name`
FROM `enrollments`
JOIN `courses` ON `courses_id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 4 APPPATH\Controllers\Auth.php(150): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-09-30 04:00:11 --> [Caused by] mysqli_sql_exception: Unknown column 'courses_name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `courses_name`
FROM `enrollments`
JOIN `courses` ON `courses_id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `courses_name`
FROM `enrollments`
JOIN `courses` ON `courses_id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `courses_name`
FROM `enrollments`
JOIN `courses` ON `courses_id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `courses_name`
FROM `enrollments`
JOIN `courses` ON `courses_id` = `enrollments`.`course_id`
WHERE `enrollments`.`student_id` = :enrollments.student_id:', [...], false)
 5 APPPATH\Controllers\Auth.php(150): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-30 04:00:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:00:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:02:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:03:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:03:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:03:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:03:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:04:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:04:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:04:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:04:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:08:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:08:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:08:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:08:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:09:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:11:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:11:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:12:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:14:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:15:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:16:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:16:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:16:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:16:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:20:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:21:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:21:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:21:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:21:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:21:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:23:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:23:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:23:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:23:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:23:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:26:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:26:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:27:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:27:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:28:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-30 04:28:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
